lib/idx/types/index.ts

```diff
@@ -67,3 +68,7 @@ export type Authenticator = {
   methodType?: string;
   phoneNumber?: string;
 };
+
+export interface ActionParameterValues {
```

Comment:
how about add `resend` to https://github.com/okta/okta-auth-js/blob/40f8bc69c0620792cc3c8451e03f5a7352cfbb6c/lib/idx/remediators/AuthenticatorEnrollmentData.ts#L18-L20? 

As we expose options type for different methods, we do want to include `resend` in the option types, so it can be accessed by devs. I know it feels a bit hacky to add value for action in the Remediator's values, but as we may restructure the idx-js response, let's put it there with a comment for future revisit.